### PR TITLE
Update Adidas icon

### PR DESCRIPTION
**Pull request by jglamp**
Adidas logo

**Issue:** helps #10282

**Popularity metric:** SimilarWeb: 1933

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Edited per #10282

### CODE REVIEW

File: icons/adidas.svg
Comment (uncenter) [inline icons/adidas.svg:1]: What are you using for your source? Grabbed this one from the header of the website and I'm seeing differences:

Screenshot 2024-05-21 at 10 22 07 (Inkscape)

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/adidas.svg
Comment (jglamp) [reply icons/adidas.svg:1]: I used the old logo that was int he SImpleIcons directory. I removed the nodes for letters in Inkscape and centered.

Comment (PratyushJayachandran) [reply icons/adidas.svg:1]: I am seeing the same
image

Please use this: `<path fill-rule="evenodd" clip-rule="evenodd" d="M 150.07 131.439 L 131.925 100 L 122.206 105.606 L 137.112 131.439 L 150.07 131.439 Z M 132.781 131.439 L 120.797 110.692 L 111.078 116.298 L 119.823 131.439 L 132.781 131.439 Z M 109.718 121.401 L 115.509 131.439 L 102.551 131.439 L 100 127.007 L 109.718 121.401 Z" fill="black"></path>` from www.adidas.com

Comment (jglamp) [reply icons/adidas.svg:1]: Can do! Apologies for the outdated logo.

Comment (uncenter) [reply icons/adidas.svg:1]: No worries. Looks like our icon was out of date in not one but two ways! Thanks for updating this.

**Comment by jglamp** [comment]
Alright, let's try this one on for size.

Adidas logo (1)

**Review by uncenter — APPROVED** [review]
Looks perfect!

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for your contribution @jglamp 🎉

**Pull request merged**